packages/atomic-react/README.md

```diff
@@ -10,6 +10,13 @@ The integration of React-based projects (using JSX) with Web based components ca
 
 `npm i @coveo/atomic-react`
 
+### Alternate installation -- CDN
```

Comment:
```suggestion
### Alternate installation — CDN
```
(you can't tell with the GitHub editor, but the dash I'm using it the length of two small dashes. It's an em dash, to be precise)